Prac_05/hex_colours.py

```diff
@@ -0,0 +1,16 @@
+"""""
+Hex Colours task CP1404
+Mischa Burgess
+"""""
+
+Colours = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "azure1": "#f0ffff", "blue1": "#0000ff",
```

Comment:
lower case variable name, or could be a constant.

Reply:
fixed it up :)